packages/http-client-java/generator/http-client-generator-test/tsp/method-override.tsp

```diff
@@ -0,0 +1,119 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+
+using Azure.Core;
+using global.Azure.Core.Traits;
+using global.Azure.Core.Foundations;
+using Azure.ClientGenerator.Core;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+
+@service({
+  title: "MethodOverride",
+})
+@versioned(Versions)
+namespace MethodOverride {
+  enum Versions {
+    @useDependency(global.Azure.Core.Versions.v1_0_Preview_2)
+    v2022_12_01_preview: "2022-12-01-preview",
+  }
+
+  model GroupQueryOptions {
+    @query foo?: string;
+    @query bar?: string;
+  }
+
+  @get
+  @route("/group-query")
+  op groupQuery is global.Azure.Core.RpcOperation<GroupQueryOptions, {}>;
+
+  model GroupAllOptions {
+    @query foo?: string;
+    @query bar?: string;
+    prop1: string;
+    prop2?: string;
+  }
+
+  @post
+  @route("/group-all")
+  op groupAll is global.Azure.Core.RpcOperation<GroupAllOptions, {}>;
+
+  model AllGroupOptions {
```

Comment:
Not used?

Reply:
Yeah, renamed to `GroupAllOptions`. Forgot to remove.